WalletWasabi.Tests/UnitTests/AllFeeEstimateTests.cs

```diff
@@ -207,6 +212,51 @@ public async Task AccurateEstimationsAsync()
 			Assert.Equal(31, allFee.Estimations[1008]);
 		}
 
+		[Fact]
+		public async Task FixObviouslyWrongEstimationsAsync()
+		{
+			var rpc = CreateAndConfigureRpcClient(
+				estimator: target => target switch
+				{
+					2 => new FeeRate(70m),
+					3 => new FeeRate(10m),
+					6 => new FeeRate(3m),
+					18 => new FeeRate(1m),
+					36 => new FeeRate(1m),
+					1008 => new FeeRate(1m),
+					_ => throw new NoEstimationException(target)
+				},
+				hasPeersInfo: true
+			);
+
+			var allFee = await rpc.EstimateAllFeeAsync(EstimateSmartFeeMode.Conservative);
+			Assert.Equal(70, allFee.Estimations[2]);
+			Assert.True(allFee.Estimations[3] > 10);
+			Assert.True(allFee.Estimations[36] > 1);
+		}
+
+
+		[Theory]
+		[Repeat(1_000)]
+		public async Task ExhaustiveEstimationsAsync(int dummy)
```

Comment:
`dummy` -> `repeatNumber` / `iterationNumber` / `iteration`?

Reply:
That's the point! I don't care because it is an unused parameter that needs to be there simply because the DataAttributte needs at least one argument (or at least I wasn't able to make it work with `Array.Empty<object>()`.

Reply:
To be honest, I would simply use a `for` cycle. `[Repeat]` attribute is elegant but it forces you to add an unused parameter and one need to guess how that `[Repeat]` actually works (sync? parallel?) or dive into `DataAttribute` implementation. Also for `[Repeat(1_000)]` it looks like it will do 1000 allocations.

Reply:
I have undone it.